Cargo.toml

```diff
@@ -58,3 +58,5 @@ tls_codec_derive  = { path = "./tls_codec/derive" }
 x509-tsp               = { path = "./x509-tsp" }
 x509-cert         = { path = "./x509-cert" }
 x509-ocsp         = { path = "./x509-ocsp" }
+
+async-signature = { git = "https://github.com/baloo/traits.git", branch = "baloo/async-signature/v0.6.0-pre.0" }
```

Comment:
wip: https://github.com/RustCrypto/traits/pull/1479

Reply:
It's released